### PR TITLE
Add -cover to test-clean-no-fdo and test-no-fdo

### PR DESCRIPTION
**Pull request by lahavyuv86**
# Description

Remove unneeded directories from our coverage and coverage-no-fdo commands.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Comment by omaciel** [comment]
@lahavyuv86  take a list of the files/folders we won't be executing tests against and add them to the `grep` section of the command to exclude them like `$(go list ./… | grep -v -e /test/ -e /unleash/)`. Make sure to only exclude files/folders we know we don't want to run tests against.

File: Makefile [hunk 1]
```diff
@@ -19,6 +19,7 @@ KUBECTL=kubectl
 NAMESPACE=default
 TEST_OPTIONS="-race"
 BUILD_TAGS=-tags=fdo
+EXCLUDE_DIRS=-e /cmd/db -e /cmd/kafka -e /cmd/migrate/orgmigration -e /config -e /pkg/clients/imagebuilder/mock_imagebuilder -e /pkg/imagebuilder/mock_imagebuilder -e /pkg/clients/inventory -e /pkg/clients/inventory/mock_inventory -e /pkg/clients/playbookdispatcher -e /pkg/errors -e /pkg/services/mock_services -e /unleash
```
Comment (omaciel) [inline Makefile:22]: I think we do want to run tests against `/cmd/migrate/orgmigration`, `/pkg/clients/playbookdispatcher`, `/pkg/clients/inventory`, no?

File: Makefile [hunk 2]
```diff
@@ -136,10 +137,10 @@ test:
 	go test $(BUILD_TAGS) $$(go list $(BUILD_TAGS) ./... | grep -v /test/) $(TEST_OPTIONS)
 
 test-clean-no-fdo:
-	go test -count=1 $$(go list ./... | grep -v /test/) $(TEST_OPTIONS)
+	go test -count=1 -cover $$(go list ./... | grep -v -e /test/ $(EXCLUDE_DIRS)) $(TEST_OPTIONS)
```
[the remark this answers is not in the document]
Comment (lahavyuv86) [reply Makefile:140]: good idea :)

File: Makefile (hunk 1 above)
Comment (lahavyuv86) [reply Makefile:22]: @ldjebran can you verify which dirs we can exclude?

**Comment by AaronH88** [comment]
We have specific `coverage` commands in this Makefile, I suggest editing them and not adding coverage to our normal test commands

**Comment by omaciel** [comment]
Thanks @AaronH88  for pointing this out. Is there a reason why we don't run that then during PR checks? I'd love to see coverage information for every new PR as it gives us information about the health of our code changes

File: Makefile (hunk 1 above)
Comment (ldjebran) [reply Makefile]: we should exclude only those that contain no tests

Comment (ldjebran) [reply Makefile]: /cmd/migrate/orgmigration  contain tests and should not be excluded

**Review by AaronH88 — CHANGES_REQUESTED** [review]
Suggest moving these changes to the `coverage` command

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Pull request merged**